<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>daojishi_Dialog</class>
 <widget class="QDialog" name="daojishi_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>同步课堂</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images/images.qrc">
    <normaloff>:/new/prefix1/Images/logo.png</normaloff>:/new/prefix1/Images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel
{
    border-style: solid;
	color:#5396ff;
	font: 75 60pt &quot;微软雅黑&quot;;
    border-width: 1px;
    border-color:#cccccc;
}
QFrame#frame{
	border-radius:10px;
	background-color: rgb(255, 255, 255);
}

QPushButton {
    border-style: solid;
    border-width: 2px;
    border-radius: 20px;

    background-color: transparent;
    background-repeat: no-repeat;
    background-position: left center;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_tmp">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/plus.png</normaloff>:/new/prefix1/Images/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tm">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_tmj">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/del.png</normaloff>:/new/prefix1/Images/del.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_mp">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/plus.png</normaloff>:/new/prefix1/Images/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_m">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_mj">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/del.png</normaloff>:/new/prefix1/Images/del.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>42</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_tsp">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/plus.png</normaloff>:/new/prefix1/Images/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ts">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_tsj">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/del.png</normaloff>:/new/prefix1/Images/del.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_sp">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/plus.png</normaloff>:/new/prefix1/Images/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_s">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>108</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_sj">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Images/images.qrc">
            <normaloff>:/new/prefix1/Images/del.png</normaloff>:/new/prefix1/Images/del.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
	border-style: solid;
	border-top-color: rgb(200,200, 200);
	border-right-color: rgb(255, 255, 255);
	border-bottom-color: rgb(255, 255, 255);
	border-left-color: rgb(255, 255, 255);
    border-width: 2px;
}
QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color: #297aff;
    border-radius: 20px;
	color:rgb(255,255,255);
	font: 16pt &quot;微软雅黑&quot;;
    background-color:#297aff;	
    background-repeat: no-repeat;
    background-position: left center;
}
</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>44</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_start">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>43</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_quxiao">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>44</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
